### PR TITLE
Add a notice for pd configuration

### PR DESCRIPTION
**Pull request by rleungx**
Signed-off-by: Ryan Leung <rleungx@gmail.com>

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Just as the title says.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: pd-configuration-file.md [hunk 1]
```diff
@@ -182,6 +182,10 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
 
 调度相关的配置项。
 
+> **注意：**
+> 
+> 通过配置文件修改这部分参数仅针对新集群有效，后续需要使用 pd-ctl 对参数的进行修改。
```
Comment (TomShawn) [inline pd-configuration-file.md:187]:
```suggestion
> 要修改调度相关的 PD 配置项 `schedule`，请根据情况使用以下一种方法：
>
> - 对于新部署集群，通过 `tiup cluster edit-config` 命令修改配置文件。
> - 对于已有集群，使用命令行工具 [PD Control](/pd-control) 进行修改。
```

Comment (rleungx) [reply pd-configuration-file.md:187]: “对于新部署集群，通过 `tiup cluster edit-config` 命令修改配置文件。” 我看 https://docs.pingcap.com/zh/tidb/dev/tiup-component-cluster-edit-config 里面描述的是 `tiup cluster edit-config` 是针对已有集群的。

Comment (TomShawn) [reply pd-configuration-file.md:187]: 调整了下 comment，PTAL

Comment (rleungx) [reply pd-configuration-file.md:187]: “要修改调度相关的 PD 配置项 `schedule`，请根据情况使用以下一种方法” -> “要修改调度相关的 PD 配置项，请根据情况使用以下一种方法”
这样如何？

Comment (TomShawn) [reply pd-configuration-file.md:187]: LGTM

**Comment by qiancai** [comment]
Removed the needs-cherry-pick-release-6.6 label because the v6.6 docs have been archived at https://docs-archive.pingcap.com/zh/tidb/v6.6 and will no longer receive new updates.

File: pd-configuration-file.md [hunk 2]
```diff
@@ -182,6 +182,13 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
 
 调度相关的配置项。
 
+> **注意：**
+> 
+> 要修改调度相关的 PD 配置项，请根据情况使用以下其中一种方法：
+>
+> - 对于新部署集群，在配置文件中修改该配置项。此方法对已有集群不生效，无法像修改配置文件中其他参数一样通过此方法修改已有集群配置文件中的 `schedule` 参数。
+> - 对于已有集群，使用命令行工具 [PD Control](/pd-control.md) 进行修改。
```
[the remark this answers is not in the document]
Comment (rleungx) [reply pd-configuration-file.md:190]: LGTM

**Review by qiancai — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

File: pd-configuration-file.md [hunk 3]
```diff
@@ -182,6 +182,13 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
 
 调度相关的配置项。
 
+> **注意：**
+> 
+> 要修改与 schedule 相关的 PD 配置项，请根据集群的情况选择以下方法之一：
```
Comment (lhy1024) [inline pd-configuration-file.md:187]: 需要强调“与 schedule 相关”么？

Comment (qiancai) [reply pd-configuration-file.md:187]: @rleungx PTAL

Comment (rleungx) [reply pd-configuration-file.md:187]: `schedule` 是个二级标题吧

Comment (qiancai) [reply pd-configuration-file.md:187]: @lhy1024 PTAL again. Thanks.

**Review by lhy1024 — APPROVED** [review]

File: pd-configuration-file.md (hunk 3 above)
Comment (qiancai) [inline pd-configuration-file.md:187]:
```suggestion
> 要修改与调度相关的 PD 配置项，请根据集群的情况选择以下方法之一：
```

File: pd-configuration-file.md [hunk 4]
```diff
@@ -182,6 +182,13 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
 
 调度相关的配置项。
 
+> **注意：**
+> 
+> 要修改与调度相关的 PD 配置项，请根据集群的情况选择以下方法之一：
+>
+> - 对于新部署集群，你可以直接在 PD 配置文件中进行修改。
+> - 对于已有集群，请使用命令行工具 [PD Control](/pd-control.md) 进行修改。直接修改 PD 配置文件中与 schedule 相关的 PD 配置项不会对已有集群生效。
```
Comment (qiancai) [inline pd-configuration-file.md:190]:
```suggestion
> - 对于已有集群，请使用命令行工具 [PD Control](/pd-control.md) 进行修改。直接修改 PD 配置文件中与调度相关的 PD 配置项不会对已有集群生效。
```

**Review by hfxsd — APPROVED** [review]

**Pull request merged**